units/en/unit7/introduction.mdx

```diff
@@ -0,0 +1,36 @@
+# Introduction [[introduction]]
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit0/thumbnail.png" alt="Thumbnail"/>
+
+Since the beginning of this course, we learned to train agents in a *single-agent system*  where our agent was alone in its environment: it was **not cooperating or collaborating with other agents**.
+
+This worked great, and the single-agent system is useful for many applications.
+
+
+<figure>
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit10/patchwork.jpg" alt="Patchwork"/>
+
+<figcaption>
+A patchwork of all the environments you’ve trained your agents on since the beginning of the course
+</figcaption>
+</figure>
+
+But, as humans, **we live in a multi-agent world**. Our intelligence comes from interaction with other agents. And so, our **goal is to create agents that can interact with other humans and other agents**.
```

Comment:
Does our intelligence really come from interaction with other agents?

Reply:
Yes, since through language we learn information from them etc. This idea is taken from mlagents blog.